docs/source/en/_toctree.yml

```diff
@@ -400,14 +400,14 @@
       title: DPMSolverSDEScheduler
     - local: api/schedulers/singlestep_dpm_solver
       title: DPMSolverSinglestepScheduler
+    - local: api/schedulers/edm_multistep_dpm_solver
+      title: EDMDPMSolverMultistepScheduler
+    - local: api/schedulers/edm_euler
+      title: EDMEulerScheduler
```

Comment:
Once https://github.com/huggingface/diffusers/pull/7329 is merged, it shouldn't be needed.